search-app/docs/api.md

```diff
@@ -1973,19 +1971,33 @@ By concept id and revision id
 
     curl -i "%CMR-ENDPOINT%/concepts/:concept-id/:revision-id"
 
-Examples:
+Plain examples, with and without revision ids:
 
     curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.iso"
-    curl -i -H 'Accept: application/xml' "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i -H 'Accept: application/metadata+xml' "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.json"
     curl -i "%CMR-ENDPOINT%/concepts/C100000-PROV1/1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1/2.echo10"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1/1"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1/2"
 
+File extension examples:
+
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.iso"
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.json"
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1/2.echo10"
+    curl -i -H "Accept: version=1.9" \
```

Comment:
Hrm... the `Accept` header is often the one used to let servers know what version to send clients ... and in fact, we do already use it this way. The only different here is that the vendor mime-type has been left out. But maybe there's a distinction I'm missing here ... good thing to chat about :-)

Regardless, I can remove that addition to the docs and use the one that you provided instead.